### ISSUE TITLE
Allow for OSC logo text customization.

### ISSUE THREAD

**Issue by stax76** [mpv-player/mpv#13125]
It's not possible to change the logo text or to remove the logo text. It would help mpv frontends with internationalization, which was recently added to mpv.net.

**Comment by stax76** [comment]
Could it even be done without config file with mpv's current capabilities? If I remember right, `user-data` can only be written to after initialization and scripts like the osc are loaded with the initialization. A creative use of an environment variable would work and be super simple and efficient.

**Comment by CogentRedTester** [comment]
Well here are the ass commands that draw the logos: https://github.com/mpv-player/mpv/blob/17be6e1990a92e9b43f5f49cca9c4dd3da24a1e8/player/lua/osc.lua#L2567-L2590

All you'd need to do is load the lines from `script-opts` or from `user-data` and you could replace it with anything.

On another note it is rather limiting to not be able to set `user-data` values from config files/profiles/the terminal.

**Comment by guidocella** [comment]
Logo customization was scrapped because you can't pass multiline strings as script-opts: https://github.com/mpv-player/mpv/pull/10207#issuecomment-1146198264

**Comment by guidocella** [comment]
They do because each line is a different ASS event: https://github.com/mpv-player/mpv/blob/17be6e1990a92e9b43f5f49cca9c4dd3da24a1e8/sub/osd_libass.c#L514

**Comment by CogentRedTester** [comment]
> They do because each line is a different ASS event:
> 
> https://github.com/mpv-player/mpv/blob/17be6e1990a92e9b43f5f49cca9c4dd3da24a1e8/sub/osd_libass.c#L512-L520

Ah you're right, my mistake. Still, it could theoretically be done by using a special substitution character that wouldn't appear inside an ass vector graphic.

**Comment by stax76** [comment]
Just hiding it, without a conf file, would be an improvement for my use case.

**Comment by CogentRedTester** [comment]
Wait are you referring to the `Drop files or URLs to play here.` text? Because *that* is one line and could easily be put into a script-opt.

**Comment by stax76** [comment]
Yes, I mean this text. I would like to have something that can be used without a file, this means script-opt might not be suitable.

**Comment by CogentRedTester** [comment]
Script-opts can be set at runtime using the `change-list` command though, the only issue I see with doing that is the possibility of the user overwriting the script opt with their own list of opts.

**Comment by stax76** [comment]
I'll try and see if it works.

**Comment by stax76** [comment]
I tried this:

CommandV("change-list", "script-opts", "append", "osc_idlescreen=no");

It does not work, probably because it runs too late, when it runs the osc was already loaded.

**Comment by guidocella** [comment]
It's `osc-idlescreen`, but there is also `script-message osc-idlescreen no`.

**Comment by stax76** [comment]
`CommandV("change-list", "script-opts", "append", "osc-idlescreen=no");`

It works! If there would be an option to hide or customize the logo text, it would probably also work.
